library/src/main/java/kwasm/ast/ValueType.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kwasm.ast
+
+enum class ValueType {
+    I32, I64, F32, F64
+}
```

Comment:
A short KDoc block on this would be good.  Also, is it worth making the values match the spec more closely?  (ie. using lowercase `i` and `f` instead of capital?)

Reply:
Code style conventions complain about lowercase letters, so I'm gonna keep the values as is. Added KDoc referencing the spec definition.

Reply:
Sounds good